test/openssl/test_ssl.rb

```diff
@@ -39,7 +39,9 @@ def test_ctx_options
   end
 
   def test_ctx_options_config
-    omit "LibreSSL and AWS-LC do not support OPENSSL_CONF" if libressl? || aws_lc?
+    if libressl? || aws_lc?
+      omit "LibreSSL and AWS-LC do not support OPENSSL_CONF"
+    end
```

Comment:
Why is this changed?

Reply:
Sorry, this change is because the line length was more than 80 bytes. I didn't like it. But I shouldn't include this change int this commit. I will revert this part.